code/script.js

```diff
@@ -1,19 +1,138 @@
+)
+
+let text;
+let NAME = prompt("Please enter your full name");
 
-// Step 1 - Welcome and introduction
-// Your code goes here
 alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+  `Thank you ` + NAME + `! Now continue to your order.`
 )
+//STEP 1 END//
+
+
+// STEP 2 - Food choice//
+// Promptmessage: Asking the user to choose a meal out of a list of options:
+const foodChoice = prompt(`Select your favourite meal!:
+  1 - Pizza
+  2 - Pasta
+  3 - Salad
+  Please enter the number of your choice:`);
+
+// Connect the customer choice to the list and response messages:
+//Initialize a variable to store the selected food.//
+let selectedChoice = "";
+
+//Determine the selected food based on the user's choice.
+if (foodChoice === "1") {
+  selectedChoice = "Pizza";
+} else if (foodChoice === "2") {
+  selectedChoice = "Pasta";
+} else if (foodChoice === "3") {
+  selectedChoice = "Salad";
+} else {
+  //Invalid choice - Errormessage:
+  alert(`Invalid choice - please select a number between 1-3 to make an order.`);
+  process.exit(1);
+}
+//Alertmessage: Confirm the selceted food option.
+alert(`You've chosen ${selectedChoice}!`);
+// STEP 2 END //
+
+
+
+// STEP 3 - Subtype choice//
+// Depending on the previously chosen food type (Pizza, Pasta, Salad), the program will present the user with subtype options specific to that food type. //
+
+let typeOfName = "";
+let subType = "";
+
+// Switch - Determine the selected subtype of food based on the user's choice.//
+//NOT SURE WHY ELSE IF IS USED ABOVE AND SWITCH HERE, BUT IT WORKS.//
+
+switch (selectedChoice) {
+  case "Pizza":
+    subType = prompt(`Select a Pizzatype:
+    1 - Margharita
+    2 - Pepperoni 
+    3 - Hawaii
+    Please enter the number of your choice:`);
+    typeOfName = ["Margharita", "Pepperoni", "Hawaii"];
+    break;
+  case "Pasta":
+    subType = prompt(`Select a Pasta type:
+    1 - Carbonara
+    2 - Mediterranian
+    3 - Bolognese
+    Please enter the number of your choice:`);
+    typeOfName = ["Carbonara", "Mediterranian", "Bolognese"];
+    break;
+  case "Salad":
+    subType = prompt(`Select a Salad:
+    1 - Cesar
+    2 - Greek 
+    3 - Fruit
+    Please enter the number of your choice:`);
+    typeOfName = ["Cesar", "Greek", "Fruit"];
+    break;
+}
+
+// Alertmessage: Confirm the selceted subType option.//
+// Had trouble understanding the typeOfName 0,1,etc.. found that others had the same issues and copied the help they got - still dont fully understand it//
+if (subType === "1") {
+  alert(
+    `Great Choice, You've chosen to order ${typeOfName[0]} ${selectedChoice}!`
+  );
+} else if (subType === "2") {
+  alert(
+    `Great Choice, You've chosen to order ${typeOfName[1]} ${selectedChoice}!`
+  );
+} else if (subType === "3") {
+  alert(
+    `Great Choice, You've chosen to order ${typeOfName[2]} ${selectedChoice}!`
+  );
+}
+else {
+  alert(`Invalid choise - please select a number between 1-3 to make an order.`)
+}
```

Comment:
I like how you organize your code!  Using subType and typeOfName(array) made your code tidier!!